sklearn/calibration.py

```diff
@@ -63,8 +63,9 @@ class CalibratedClassifierCV(BaseEstimator, ClassifierMixin):
         - An iterable yielding train/test splits.
 
         For integer/None inputs, if ``y`` is binary or multiclass,
-        :class:`StratifiedKFold` used. If ``y`` is neither binary nor
-        multiclass, :class:`KFold` is used.
+        :class:`sklearn.model_selection.StratifiedKFold` is used. If ``y`` 
```

Comment:
It's not enough to generate the link in the current version of the docs for [CalibratedClassifierCV](http://scikit-learn.org/stable/modules/generated/sklearn.calibration.CalibratedClassifierCV.html). 

It works in `cross_val_predict`, but that class is in the same module as `StratifiedKFold`. 

Full import is used elsewhere in the documentation, e.g., in `/preprocessing/data.py`:
(e.g. as part of a preprocessing :class:`sklearn.pipeline.Pipeline`)
to give the correct link, as seen [here](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.scale.html).